content/riak/cs/2.1.2/cookbooks/release-notes.md

```diff
@@ -2,36 +2,142 @@
 title: "Riak CS Release Notes"
 description: ""
 menu:
-  riak_cs-2.1.2:
+  riak_cs-3.0.0:
```

Comment:
Isn't this Riak CS 2.1.2's release notes? Not 3.0.0?

Reply:
whoops!